client/asset/btc/btc.go

```diff
@@ -486,8 +486,8 @@ type WalletConfig struct {
 	FallbackFeeRate  float64 `ini:"fallbackfee"`
 	FeeRateLimit     float64 `ini:"feeratelimit"`
 	RedeemConfTarget uint64  `ini:"redeemconftarget"`
-	ActivelyUsed     bool    `ini:"special:activelyUsed"` // injected by core
-	Birthday         uint64  `ini:"walletbirthday"`       // SPV
+	ActivelyUsed     bool    `ini:"specialactivelyUsed"` // injected by core
```

Comment:
The colon no longer works in the ini package for some reason:
https://github.com/decred/dcrdex/commit/2b4ce79c61302044a045b3fb5a85fcf6b71fa2e6